pkg/trace/agent/fuzz_test.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build go1.18
+//go:build go1.18 && disabled
```

Comment:
```suggestion
//go:build go1.18 && !windows
```
As a workaround to https://github.com/golang/go/issues/52268 - the [fix](https://github.com/golang/go/commit/c3470ca83cfdbcc71ce162b00d6b9a0004a7f649) is in 1.19